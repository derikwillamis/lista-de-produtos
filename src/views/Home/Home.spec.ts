import {render} from '@testing-library/vue'
import Component from './Home.vue'

describe('Home', () => {
  describe('render', () => {
    it('renders correctly', () => {
      const {container} = render(Component)
      expect(container).toMatchSnapshot('Home')
    })
  })
})

