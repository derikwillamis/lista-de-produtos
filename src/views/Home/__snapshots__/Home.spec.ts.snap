// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home render renders correctly: Home 1`] = `
<div>
  <div>
    <div
      class="header"
    >
      <div>
        <input
          data-testid="input"
          placeholder="Buscar por titulo"
          type="text"
        />
      </div>
       
      <div>
        <select
          id="select"
        >
          <option
            value=""
          >
            
      Todos
    
          </option>
          <option
            value="processing"
          >
            
      Processando
    
          </option>
          <option
            value="processed"
          >
            
      Solicitada
    
          </option>
          <option
            value="created"
          >
            
      Conclu√≠da
    
          </option>
        </select>
      </div>
    </div>
     
    <div
      class="container"
    >
      <div
        class="loading"
      />
       
      <!---->
    </div>
     
    <!---->
  </div>
</div>
`;
